tfhe/src/integer/parameters/mod.rs

```diff
@@ -117,3 +119,44 @@ pub const PARAM_MESSAGE_1_CARRY_1_KS_PBS_32_BITS: WopbsParameters = WopbsParamet
     ciphertext_modulus: CiphertextModulus::new_native(),
     encryption_key_choice: EncryptionKeyChoice::Big,
 };
+
+pub struct CiphertextConformanceParams {
+    pub shortint_params: shortint::parameters::CiphertextConformanceParams,
+    pub num_blocks_per_integer: usize,
+}
+
+/// Structure to store the expected properties of a ciphertext
+/// Can be used on a server to check if client inputs are well formed
+/// before running a computation on them
+impl CiphertextConformanceParams {
+    pub fn to_ct_list_conformance_parameters(
+        &self,
+        list_constraint: ListSizeConstraint,
+    ) -> CiphertextListConformanceParams {
+        CiphertextListConformanceParams {
+            shortint_params: self.shortint_params,
+            num_blocks_per_integer: self.num_blocks_per_integer,
+            num_integers_constraint: list_constraint,
+        }
+    }
+}
+
+/// Structure to store the expected properties of a ciphertext list
+/// Can be used on a server to check if client inputs are well formed
+/// before running a computation on them
+pub struct CiphertextListConformanceParams {
```

Comment:
If we can have the Compact naming if those are destined to be used with compact ciphertexts please (same for all structs related to compact ciphertexts)

Reply:
It don´t think there's anything specific to compact (but to radix yes)

Reply:
aren't those used for the compact ciphertexts ? I may I have misread I just don't recall CiphertextLists other than compact ones at the integer level 🤔

Reply:
Indeed, we don't have any other lists (I assume we did)
But there is nothing specific to compact in it anyway

Reply:
Ok, I still would like to have the compact naming as for now it corresponds to the compact part and it must be checking the compact LWE ciphertext list right ? This is not the expanded version ?

Reply:
It is currently only used for Compact
I'll rename it :)

Reply:
Done